### ISSUE TITLE
Large mayority of sequences currently assigned to BA.2.12 doesn't carry defining mutation S:704L 

### ISSUE THREAD

**Issue by FedeGueli** [cov-lineages/pangolin#437]

**Comment by corneliusroemer** [comment]
This is a pangoLEARN issue, not a designation issue

**Comment by FedeGueli** [comment]
Thank you Cornelius

**Comment by aineniamh** [comment]
This is a designation issue @corneliusroemer as its very likely to have been caused by designated sequences with ambiguities at that site. The solution will be to remove those designated sequences.

**Comment by AngieHinrichs** [comment]
I agree it's a designation issue but I think the main problem for BA.2.12 is that it's part of a large undesignated clade BA.2 + C25416T, and pangoLEARN needs sequences with C25416T but without S:704L to be designated as BA.2 so it doesn't get the false impression that C25416T only occurs in BA.2.12.

**Comment by FedeGueli** [comment]
@AngieHinrichs thank you ! Being 96% of sequences C25416T actually assigned to BA.2.12,  why not broademing BA.2.12 to C25416T ?
It wont change anything for other sublineages and notvhing would change with BA.2.12.1 (sublineage of interest). Thus broadening BA.2.12 starting with C25416T willl permit to designate correctly any new sublineage with epi relevance  popping out from that branch.
cc @corneliusroemer
